tests/bluetooth/common/testlib/src/att_read.c

```diff
@@ -167,17 +174,16 @@ int bt_testlib_gatt_long_read(struct net_buf_simple *result_data, uint16_t *resu
 {
 	int err;
 	uint16_t _result_data_size = 0;
-
-	struct bt_testlib_att_read_closure ctx = {
-		.long_read = true,
-		.result_size = &_result_data_size,
-		.conn = conn,
-		.result_data = result_data,
-		.params = {
-			.handle_count = 1,
-			.single = {.handle = handle, .offset = offset},
-			IF_ENABLED(CONFIG_BT_EATT, (.chan_opt = bearer,))
-		}};
+	struct bt_testlib_att_read_closure ctx = {};
+
+	ctx.att_mtu = result_att_mtu;
```

Comment:
It broke bbsim CI https://github.com/zephyrproject-rtos/zephyr/actions/runs/7099469135/job/19323552333?pr=66136#step:12:997

Reply:
fixing it in https://github.com/zephyrproject-rtos/zephyr/pull/66012
feel free to cherry-pick

Reply:
@alwa-nordic Ping.

Reply:
OK, thanks.